src/clj/kixi/hecuba/api/measurements.clj

```diff
@@ -34,6 +34,12 @@
 
 (def ^:private uploads-status-resource-path (p/resource-path-string :uploads-status-resource))
 
+(defn get-measurements-keys [sensor-keys session]
```

Comment:
Does `keys` mean `ids` here? I'm not sure of what are measurements keys or sensor keys.